core/math/color.cpp

```diff
@@ -595,6 +595,9 @@ Color Color::from_ok_hsl(float p_h, float p_s, float p_l, float p_alpha) {
 }
 
 float Color::get_ok_hsl_h() const {
+	if (r == 0.0 && g == 0.0 && b == 0.0) {
+		return 0.0;
```

Comment:
Same here (except in this case, the compiler knows you are making a mistake and corrects it).